src/processing/script/instrumented.js

```diff
@@ -65,6 +65,13 @@ export const PROPERTIES = [
 const INSTRUMENTED_METHOD_RE   = new RegExp(`^(${METHODS.join('|')})$`);
 const INSTRUMENTED_PROPERTY_RE = new RegExp(`^(${PROPERTIES.join('|')})$`);
 
+// NOTE: The obfuscated version of the mootools framework contains code
+// that removes the RegExp.prototype.test method and restores it later.
+//    delete z[A]; // z = RegExp.prototype, A = "test"
+//    __set$(z, A, x.protect()); // x.protect - returns the removed method
+// The __set$ function calls the test method of the regular expression. (GH-331)
+var test = Function.prototype.call.bind(RegExp.prototype.test);
```

Comment:
Let's use arrow here with saved proto.test instead of 'bind'. Just found out that 'bind' can be extremely slow due to additional semantics.